$OpenBSD: patch-editor_dconf-editor_desktop_in,v 1.1 2011/09/27 15:01:54 ajacoutot Exp $

There is no "dconf-editor" icon yet, so use a gtk stock one.

--- editor/dconf-editor.desktop.in.orig	Tue Sep 27 16:57:09 2011
+++ editor/dconf-editor.desktop.in	Tue Sep 27 16:57:28 2011
@@ -4,7 +4,7 @@ Comment=Directly edit your entire configuration databa
 Exec=dconf-editor
 Terminal=false
 Type=Application
-Icon=dconf-editor
+Icon=gtk-preferences
 StartupNotify=true
 Categories=GNOME;GTK;System;
 X-GNOME-Bugzilla-Bugzilla=GNOME
